pkg/cmd/cli/cmd/set/imagelookup.go

```diff
@@ -250,7 +311,24 @@ func (o *ImageLookupOptions) printImageLookup(infos []*resource.Info) error {
 	defer w.Flush()
 	fmt.Fprintf(w, "NAME\tLOCAL\n")
 	for _, info := range infos {
-		fmt.Fprintf(w, "%s\t%t\n", info.Name, info.Object.(*imageapi.ImageStream).Spec.LookupPolicy.Local)
+		switch t := info.Object.(type) {
+		case *imageapi.ImageStream:
+			fmt.Fprintf(w, "%s\t%t\n", info.Name, t.Spec.LookupPolicy.Local)
+		default:
+			accessor, ok := ometa.GetAnnotationAccessor(info.Object)
+			if !ok {
+				fmt.Fprintf(w, "%s/%s\tUNKNOWN\n", info.Mapping.Resource, info.Name)
```

Comment:
1. Make this error message more readable, `UNKNOWN` is not helpful.
2. This method allows returning errors, but you're not actually using it anywhere, it would be good to either gather errors in a list and return aggregate, or return at any error. I'd personally went with the former.